src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/response/NodeSummaryResponse.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.response;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NodeSummaryResponse {
+  private String nodeId;
+  private String ipAddress;
+  private List<ResourceSummaryResponse> resourceList;
```

Comment:
Can we just use List of Record here instead of converting it to another object ResourceSummaryResponse ? Record contains all information we need (name of each field / value of each field) and we simply need to add a function to serialize the Record to String.

Reply:
To convert the Record into JsonResponse, we need to group resources for all nodes and all rca together. We can either use a bunch of HashMaps or have separate classes for that. I prefer this way as it makes it more readable. Let me know what you think